tools/README.md

```diff
@@ -27,16 +23,15 @@ to test.
 
 The script `prepare_for_prebuilt_workers.go` takes the following options:
 * `-l `<br> Language and GITREF to benchmark. The language and its specific 
-GITREF wish to build workers from can be specified as `language:gitref`.
+GITREF wish to build workers from can be specified as `language:COMMIT_SHA`.
```

Comment:
Nit: I think the word "wish" makes it sound like we're introducing a new clause. We can just omit it. "from" gets the point across. 

```suggestion
GITREF to build the workers from can be specified as `language:COMMIT_SHA`.
```